core/dbt/task/base.py

```diff
@@ -192,6 +192,20 @@ def __init__(self, node):
         self.node = node
 
 
+def skip_result(node, message):
```

Comment:
Nitty nits: This is from previous code, but `skip_result` function name seems like an action instead of result, wanna consider updating it to something along the line of `skipped_result` or `result_skipped`?